### ISSUE TITLE
Is this project maintained?

### ISSUE THREAD

**Issue by bscott** [muesli/beehive#390]
Does this project need more maintainers to assist with issue handling and merging of PR's

**Comment by muesli** [comment]
@bscott Yes, it is, but it's currently undergoing a major upgrade/rewrite. Feel invited to join [our chat](https://matrix.to/#/#beekeepers:matrix.org) if you want to find out more, or just wait a couple more days for some news!

**Comment by bscott** [comment]
Thanks @muesli Happy to contribute as well :)

**Comment by MatthewAry** [comment]
Out of curiosity, will the rewrite have the same license?

**Comment by muesli** [comment]
No, we're actually going to switch to MIT.

**Comment by ddemoss222** [comment]
I am excited to see some of the changes. Any updates on this?

**Comment by helmut72** [comment]
> or just wait a couple more days for some news!

Any news?

**Comment by fr3fou** [comment]
any news? :D

**Comment by dbgithub76** [comment]
No further updates.  Didn’t continue along this line.  Thanks for asking though.  --------David BurrowsOn 4 Dec 2022, at 21:31, Simo Aleksandrov ***@***.***> wrote:﻿
any news? :D

**Comment by bscott** [comment]
Should we check with https://matrix.to/#/#beekeepers:matrix.org, its been a few months, just checking if the rewrite is still active, happy to contribute @muesli

**Comment by smyja** [comment]
> Should we check with https://matrix.to/#/#beekeepers:matrix.org, its been a few months, just checking if the rewrite is still active, happy to contribute @muesli 

Seems the project isn't active.

**Comment by pombredanne** [comment]
@muesli gentle ping. What's up and what's next? :P 👼
